source/storage.rst

```diff
@@ -0,0 +1,155 @@
+use as any other disk from within the VM.
+
+Management
+::::::::::
+
+OpenStack providers offer block storage using their native API. Both the
+CLI client or dashboard can be used. Main commands for managing volumes are
+listed below:
+
++--------------------------------------------+--------------------------------------+
+| Command                                    | function                             |
++============================================+======================================+
+| ``volume create --size <size> <name>``     | create a volume of size ``<size>``   |
+|                                            | GBs and name ``<name>``              |
++--------------------------------------------+--------------------------------------+
+| ``volume list``                            | list available volumes               |
++--------------------------------------------+--------------------------------------+
+| ``volume show <volume>``                   | show details of a given volume       |
++--------------------------------------------+--------------------------------------+
+| ``volume delete <volume>``                 | deletes a volume                     |
++--------------------------------------------+--------------------------------------+
+| ``server add volume <server> <volume>``    | Attach a volume to a server          |
++--------------------------------------------+--------------------------------------+
+| ``server remove volume <server> <volume>`` | Dettach a volume from a server       |
++--------------------------------------------+--------------------------------------+
+
+For using the legacy OCCI interface refer to `OCCI How-to <https://wiki.egi.eu/wiki/HOWTO11_How_to_use_the_rOCCI_Client>`_.
+
+Using block storage from your VMs
+:::::::::::::::::::::::::::::::::
+
+Block Storage will appear as block devices into your VM. Usually these devices
+are empty upon creation. The first time you attach them to a VM, you will need
+to partition and create filesystems on the device.
+
+You can just create a filesystem on the block device with the following
+command (run this at your VM!).
+
+.. warning::
+
+   Only run this command the first time you use the device, it removes all data storedi
+
+.. code-block:: console
+
+   # mkfs.ext4 /dev/<volume device>
+
+The ``<volume device>`` (e.g ``vdb``) can be obtained with the ``openstack volume show``
+command
+
+Once you have a filesystem you can mount it at the desired path:
+
+.. code-block:: console
+
+   # mount /dev/<volume device> /<path>
+
+With that you can access ``/<path>`` where all your data will be available.
+Applications will not see any difference between a block storage device and a
+regular hardware disk, thus no major changes should be required in the application logic.
+
+Note that some OS, like `CERNVM <https://appdb.egi.eu/store/vappliance/cernvm/ CERNVM>`_,
+will automatically detect all the attached block storage and add it to theo
```

Comment:
```suggestion
will automatically detect all the attached block storage and mount it under the
```
Not sure about the intended meaning, it may seem a bit obscure.

Reply:
removed the whole paragraph, not really bringing much